### DISCUSSION TITLE
MACBOOK PRO M2 CHIP

### DISCUSSION THREAD

**Discussion by ccgr33n** [ExistentialAudio/BlackHole#711] [Q&A]
This would be Ventura and M2. - will blackhole work? Old Macbook is being replaced by Apple, so no choice. Thank you

**Comment by devinroth** [comment]
Works on Apple Silicon. Haven’t specified testing on M2 but I can’t see why it wouldn’t work.

Devin

**Comment by ccgr33n** [comment]
Much appreciated.
Will I need to update or reinstall blackhole driver.
If so, is there a link?

Thanks again.

Stay safe. Remain healthy. Enjoy life.


*~* debra
Debra Menaker
***@***.***>

***@***.*** ***@***.***>

**Comment by devinroth** [comment]
Use brew.

Devin

**Comment by ccgr33n** [comment]
Thank you for the quick response.
Don't believe I understand:
Use brew.
Thanks again.

*~* debra
Debra Menaker
***@***.***>

***@***.*** ***@***.***>

**Comment by devinroth** [comment]
In terminal “brew install BlackHole-2ch”

Devin

**Reply by ccgr33n** [reply]
Really appreciate your time and information

**Comment by LaKing** [comment]
It works on M2, ...
